sdks/python/apache_beam/examples/streaming_wordcount.py

```diff
@@ -25,16 +25,19 @@
 
 import argparse
 import logging
-import re
 
 
 import apache_beam as beam
 import apache_beam.transforms.window as window
 
 
+def split_fn(lines):
+  import re
+  return re.findall(r'[A-Za-z\']+', x)
```

Comment:
I see this as a temporary change until we can use the save_main_session flag in streaming mode. We can revert back this changes once that happens.